.github/workflows/ci.yml

```diff
@@ -89,11 +89,15 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           args: --all-targets ${{ matrix.features }} ${{ matrix.build }} -- -D warnings
+        env:
+          CARGO_TARGET_DIR: /home/runner/work/geoengine/geoengine/target
       - name: Run tests
         uses: actions-rs/cargo@v1
         with:
           command: test
           args: ${{ matrix.features }} ${{ matrix.build }} --verbose
+        env:
+          CARGO_TARGET_DIR: /home/runner/work/geoengine/geoengine/target
```

Comment:
is this necessary or was it just the cache?

Reply:
The problem is that utoipa packages the UI part of the SwaggerUI and the build.rs of utoipa moves this into a "wrong" folder. The folder where it is moved is not covered by the cache. However, utoipa tries to access the UI files and this crashes the test/build process.
Using the CARGO_TARGET_DIR we can force cargo to move all build dependecies into a specific folder. And the build.rs of utoipa also uses this env variable to determine the folder whrere the ui stuff is moved into.

Reply:
Can you link the utoipa issue as a TODO?

Reply:
Could you create one and link it?

Reply:
unfortunately I can't, because I don't understand the problem well enough. Johannes would have to do that.

Reply:
Since he already commented, let's include him again?

Reply:
4e2e5fddb02b1e8a80060d89894e7261adc621cf